app/actions/remote/post.ts

```diff
@@ -267,8 +302,26 @@ export const fetchPosts = async (serverUrl: string, channelId: string, page = 0,
     }
 
     try {
-        const data = await client.getPosts(channelId, page, perPage);
-        return processPostsFetched(serverUrl, ActionType.POSTS.RECEIVED_IN_CHANNEL, data, fetchOnly);
+        const isCRTEnabled = await getIsCRTEnabled(operator.database);
+        const data = await client.getPosts(channelId, page, perPage, isCRTEnabled, isCRTEnabled);
+        const result = await processPostsFetched(serverUrl, ActionType.POSTS.RECEIVED_IN_CHANNEL, data, true);
+        if (!fetchOnly) {
+            const models = await operator.handlePosts({
+                ...result,
+                actionType: ActionType.POSTS.RECEIVED_SINCE,
+                prepareRecordsOnly: true,
+            });
+            if (isCRTEnabled) {
```

Comment:
This seems error prone for the caller of this function.

I may call fetchPosts as part of a more complex action and want to batch the results. It is easy to forget that we have to process the posts and process the threads. I wonder if this should be inside the handlePosts function.

Reply:
I would leave `handlePosts` as is and have `handleThreads` as is. the way forward here is that the return statement should include both so that the caller does not forget.